lib/Core/Executor.cpp

```diff
@@ -1526,6 +1526,79 @@ void Executor::executeInstruction(ExecutionState &state, KInstruction *ki) {
     }
     break;
   }
+  case Instruction::IndirectBr: {
+    // implements indirect branch to a label within the current function
+    const auto bi = cast<IndirectBrInst>(i);
+    auto address = eval(ki, 0, state).value;
+    address = toUnique(state, address);
+
+    // concrete address
+    if (const auto CE = dyn_cast<ConstantExpr>(address.get())) {
+      const auto bb_address = (BasicBlock *) CE->getZExtValue(Context::get().getPointerWidth());
+      transferToBasicBlock(bb_address, bi->getParent(), state);
+      break;
+    }
+
```

Comment:
I believe that it could be cleaner to have an `if-else` and a single `break` at the end of the `case`.

Reply:
That would increase the nesting of 60 lines. I prefer early exists (and so does the [LLVM coding standard](https://llvm.org/docs/CodingStandards.html#use-early-exits-and-continue-to-simplify-code)).

Reply:
@251 yeah, it is more a personal style comment.